gooddata-pandas/gooddata_pandas/data_access.py

```diff
@@ -16,6 +16,7 @@
     ObjId,
     TableDimension,
 )
+from gooddata_sdk.utils import IdObjType
```

Comment:
Is this import correct?

Reply:
Yes, before that [catalog.find_label_attribute](https://github.com/hkad98/gooddata-python-sdk/blob/3bee712c76dced06be8e46264a12218306cf1af9/gooddata-sdk/gooddata_sdk/catalog/workspace/entity_model/content_objects/dataset.py#L166-L166) was used and based on that typing the same import was used [see](https://github.com/hkad98/gooddata-python-sdk/blob/3bee712c76dced06be8e46264a12218306cf1af9/gooddata-sdk/gooddata_sdk/catalog/workspace/entity_model/content_objects/dataset.py#L17-L17).